frontend/public/components/markdown-view.tsx

```diff
@@ -89,13 +89,19 @@ export class SyncMarkdownView extends React.Component<
         <link rel="stylesheet" href="${link.href}">`,
       '',
     );
+    const content = this.props.truncateContent
+      ? _.truncate(this.props.content, {
```

Comment:
It would be better to use CSS line clamping. See the `co-line-clamp` mixin.

Reply:
@spadgett  I've already tried this approach, by adding `co-line-clamp` to the main div.
![Screenshot 2020-06-26 at 12 52 21](https://user-images.githubusercontent.com/1668218/85849942-29194000-b7ac-11ea-81e8-070392546c3a.png)

The issue is with the multiple paragraph blocks

<img width="292" alt="Screenshot 2020-06-26 at 12 47 47" src="https://user-images.githubusercontent.com/1668218/85850031-4ea64980-b7ac-11ea-86e9-d6ed4b2b941d.png">

For that reason I went with the programatic solution here. Other solution would be to tweak the CSS for the markdown iframe, but would rather confirm that approach before starting with the work.